train_configs/debug_model.toml

```diff
@@ -33,7 +33,7 @@ batch_size = 8
 seq_len = 2048
 warmup_steps = 2  # lr scheduler warm up, normally 20% of the train steps
 max_norm = 1.0  # grad norm clipping
-steps = 10
+steps = 100
```

Comment:
Are these debug artifacts?

Reply:
Thanks for the review. The requested changes are made.